ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -516,6 +784,19 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-upi
+- as: azure-upi-mini-perm-arm-f28
```

Comment:
If we add this new profile, we need to remove the above one - `azure-upi-arm`

Reply:
sure